### ISSUE TITLE
Error: connect ECONNREFUSED 127.0.0.1:8001

### ISSUE THREAD

**Issue by dragonfire1119** [aquafadas-com/gulp-php-minify#2]
When I run ``gulp default`` it does this. After a few tries it might work.

**Comment by cedx** [comment]
Thanks you for reporting this issue. This was fixed in version 1.0.1 :
https://github.com/cedx/gulp-php-minify/issues/1

Just upgrade your `package.json` file with the latest package version :
```json
{
  "dependencies": {
    "@cedx/gulp-php-minify": "^1.1.0"
  }
}
````

Beware of the breaking changes:
https://github.com/cedx/gulp-php-minify/blob/master/CHANGELOG.md

Please read carefully the documentation:
https://www.npmjs.com/package/@cedx/gulp-php-minify

**Comment by dragonfire1119** [comment]
@cedx Thanks for the response. I guess there is a new repo? I have downloaded this new repo & it's not minifying anything it says it is but I don't see any dist folder. Thanks for your time!

**Comment by cedx** [comment]
Yes, there's a new repo here: https://github.com/cedx/gulp-php-minify

I need more details: your Node.js version, your PHP version (and path ?), the Gulp-Smartling plugin version, and every detail that could help me to investigate.

**Comment by dragonfire1119** [comment]
NPM: 3.10.8
PHP: 7.1.0

I'm not use Gulp-Smartling?

**Comment by cedx** [comment]
Sorry, a bad copy paste (off course this is not Gulp-Smartling, but Gulp-PHP-Minify).

I've released a new version a new version 2.0.0 that should fix this issue :
https://www.npmjs.com/package/@cedx/gulp-php-minify

See the updated [documentation](https://github.com/cedx/gulp-php-minify/blob/master/README.md) about the new `mode` option that let you select between "a slow thing that works everytime", and "a fast thing that fails most of the time".

**Comment by cedx** [comment]
@alphayax Could you close this issue? Thanks in advance.
